packages/manager/src/managers/project/ProjectManager.ts

```diff
@@ -193,6 +193,27 @@ export class ProjectManager extends BaseManager {
 		return path.dirname(sliceMachinePackageJSONPath);
 	}
 
+	async getAdapterName(): Promise<string> {
+		const sliceMachineConfig = await this.getSliceMachineConfig();
+		const adapterName =
+			typeof sliceMachineConfig.adapter === "string"
+				? sliceMachineConfig.adapter
+				: sliceMachineConfig.adapter.resolve;
+
+		return adapterName;
+	}
+
+	async locateAdapterUIDir(): Promise<string> {
```

Comment:
This method might be better named `locateAdapterDir`. The similar method named `locateSliceMachineUIDir` is named such because it locates the `slice-machine-ui` package.

Reply:
Done, that's a mistake on my side, was too fast here, thank you 🙂